package.json

```diff
@@ -320,6 +320,7 @@
     "supports-color": "3.1.2"
   },
   "devDependencies": {
+    "@coderbyheart/karma-sauce-launcher": "coderbyheart/karma-sauce-launcher#5259942cd6d40090eaa13ceeef5b0b8738c7710f",
```

Comment:
Looking forward to switching to our fork, although I have some questions I need to get on there and ask about that too...